src/clients/rust/Cargo.toml

```diff
@@ -2,14 +2,30 @@
 name = "tigerbeetle"
 version = "0.1.0"
```

Comment:
We set version during publishing, so I am wondring if we can set this one to `0.0.0`, to make it clear that this is not a real version?